src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -385,12 +391,12 @@ protected PomLabel getPomLabel() {
     }
 
 
-    protected AuthConfig prepareAuthConfig(ImageName image, String configuredRegistry, boolean isPush)
+    protected AuthConfig prepareAuthConfig(ImageName image, String configuredRegistry, boolean isPush, boolean skipExchange)
```

Comment:
can we use `skipEchange` directly from member var ? This wouldn't pollute the the argument list that much (which always tend to grow unbounded ;)